Extension/src/Debugger/extension.ts

```diff
@@ -38,8 +38,18 @@ export function initialize() {
     disposables.push(vscode.window.onDidChangeActiveTextEditor(onDidChangeActiveTextEditor));
     onDidChangeActiveTextEditor(vscode.window.activeTextEditor);
 
-     // Activate Adapter Commands 
-     registerAdapterExecutableCommands();
+
+    // Activate Adapter Commands 
+    registerAdapterExecutableCommands();
+
+    const documentSelector: vscode.DocumentSelector = [{
```

Comment:
add comment and make sure you put the getconfigsnippets near here since that has to happen first.